app/views/manuals/_header.html.erb

```diff
@@ -28,16 +28,16 @@
   </div>
 </header>
 <ol class='breadcrumb-trail'>
-  <% if request.path == manual.url %>
+  <% if request.path == presented_manual.url %>
     <li class="no-separator">Contents</li>
   <% else %>
-    <li><%= link_to 'Contents', manual.url %></li>
+    <li><%= link_to 'Contents', presented_manual.url %></li>
   <% end %>
 
-  <% if @document %>
-    <% @document.breadcrumbs.each do | crumb | %>
+  <% if defined?(document) %>
```

Comment:
Minor and it's up to you if you want to change, but I've always preferred `local_assigns.has_key?  :document` to `defined?(document)` for these cases.